### ISSUE TITLE
Multiple outputs do not work

### ISSUE THREAD

**Issue by Mehonoshin** [dwbutler/logstash-logger#30]
Hi!

I'm trying to use logstash-logger to send my rails logs to remote Logstash server and to write them to file, as usual.

I've added to application.rb

```
    config.logstash = [
      {
        type: :file,
        path: 'log/production.log'
      },
      {
        type: :tcp,
        port: 5000,
        host: 'localhost'
      }
    ]
```

And receive

13:07:04 worker.1 | no implicit conversion of Symbol into Integer
13:07:04 worker.1 | /Users/smekhonoshin/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/logstash-logger-0.7.0/lib/logstash-logger/device.rb:31:in `[]'
13:07:04 worker.1 | /Users/smekhonoshin/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/logstash-logger-0.7.0/lib/logstash-logger/device.rb:31:in `parse_uri_config'
13:07:04 worker.1 | /Users/smekhonoshin/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/logstash-logger-0.7.0/lib/logstash-logger/railtie.rb:9:in `setup'
13:07:04 worker.1 | /Users/smekhonoshin/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/logstash-logger-0.7.0/lib/logstash-logger/railtie.rb:33:in `block in <class:Railtie>'

Does anybody know how to solve this issue?

**Comment by dwbutler** [comment]
This feature has been merged but not released yet. If you want to try it out, you can link to the Github repo.

**Comment by Mehonoshin** [comment]
@dwbutler thanks, I'll check it out.
And another question, is it possible to write to file raw rails logs and send to logstash their json representation?
For development it is more pretty to see them in original format.

**Comment by kikicarbonell** [comment]
@dwbutler, I have the same problem on 0.7.0 gem from rubygems.org; I thinks that you need remove it from the oficial docs of 0.7.0 version.

**Comment by dwbutler** [comment]
I don't think it's possible to do that from a single `Logger` instance. The logger will first format the log message into a string using the formatter, and then send the formatted output to the device.

You should probably just not use LogStashLogger in development. =)

**Comment by dwbutler** [comment]
This feature has been released in version 0.8.0. Sorry again for the confusion.
